libs/ui-toolkit/src/components/sparkline/sparkline.tsx

```diff
@@ -43,6 +43,8 @@ export const SparklineView = ({
 
   // Get the extent for our y value
   const [min, max] = extent(marketData, (d) => d[1]);
+  console.dir(marketData.length);
```

Comment:
chore (blocking): stray logging

Reply:
What's even worse was.. nothing else in this file changed 🤦 I've reverted this file.